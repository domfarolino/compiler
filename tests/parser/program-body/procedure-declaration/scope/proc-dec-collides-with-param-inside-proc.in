program dominic is
  procedure function1(integer arg1 in, float arg2 in, bool arg3 in, string arg4 out, char arg5 inout)

    procedure arg1()
    begin
    end procedure;

  begin
  end procedure;

begin
end program.
