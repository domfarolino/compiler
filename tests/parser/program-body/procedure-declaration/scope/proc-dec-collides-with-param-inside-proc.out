[1;31mLine 5: Symbol 'arg1' already exists, cannot redeclare
Line 5: Error parsing program body
[0m